src/Numerics/Ylm.h

```diff
@@ -127,5 +127,44 @@ inline void derivYlmSpherical(const int l,
   }
 }
 
+// wrapper for Ylm, which doesn't require a unit vector and is in the normal order (x,y,z)
```

Comment:
In the function which needs (x,y,z) ordering, please add a note in the function documentation.
Here specify the ordering needed by this function.

Reply:
added and updated comments. Should be more clear